modules/nf-core/merfin/hist/main.nf

```diff
@@ -0,0 +1,62 @@
+process MERFIN_HIST {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/merfin:1.0--h4ac6f70_2':
+        'biocontainers/merfin:1.0--h4ac6f70_2' }"
+
+    input:
+    tuple val(meta), path(fasta_assembly)   // Required Input -sequence files can be FASTA or FASTQ; uncompressed, gz compressed.
+    tuple val(meta1), path(meryl_db_reads)  // Required readmers (raw reads meryl db). As it comes from another tool, it might be relevant to mantain the meta.
```

Comment:
It's an option. I do think that using meta1 can simplify thinks and make the code clean because in second channel (`tuple val(meta1), path(meryl_db_reads)`) we could directly use the output channel `meryl_db` from module [MERYL_HISTOGRAM](https://nf-co.re/modules/meryl_histogram). Let me know if you have another opinion and thank you very much for your reviews.

Reply:
Additional comment: I'll try to fix the `nf-test` failures in conda and singularity environment.

Reply:
It's slightly controversial and both approaches have some rationale behind them. As long as you have an idea of how it will be integrated in pipelines, I don't see much issue with having two channels if it makes things simpler.